src/test/resources/pages/instructorCourseEnrollError.html

```diff
@@ -60,7 +60,11 @@
                     </span>
                     <br>
                     <span class="problemDetail">
-                      • "bjack.gmail.tmt" is not acceptable to TEAMMATES as an email because it is not in the correct format. An email address contains some text followed by one '@' sign followed by some more text. It cannot be longer than 254 characters. It cannot be empty and it cannot have spaces.
+                      •
+                      <b>
+                        Email
+                      </b>
+                      must contain some text followed by one '@' sign followed by some more text. It cannot be longer than 254 characters and cannot have spaces.
```

Comment:
Looks like we are losing some information here. It is useful to show which value is the problematic one.